<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>541</width>
    <height>552</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Command Launcher</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Applications registry</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="cmd_input">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>461</width>
      <height>23</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Enter commands to be run.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_run">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Run selected command in the command registry.</string>
    </property>
    <property name="text">
     <string>Run</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_add">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>50</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Add new command to command registry.</string>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_remove">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>50</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Remove selected command from the command registry.</string>
    </property>
    <property name="text">
     <string>Remove</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Command Input</string>
    </property>
   </widget>
   <widget class="QListWidget" name="cmd_registry">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>451</width>
      <height>291</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Double click to run any selected command or click at the button run.</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chb_editable">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>50</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>If checked, any item in the command registry become editable instead of being run.</string>
    </property>
    <property name="text">
     <string> Commands Editable</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chb_always_on_top">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>70</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>If checked, any item in the command registry become editable instead of being run.</string>
    </property>
    <property name="text">
     <string>Always on top</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>541</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
